networkx/algorithms/flow/maxflow.py

```diff
@@ -452,7 +446,7 @@ def minimum_cut(flowG, _s, _t, capacity="capacity", flow_func=None, **kwargs):
     if not callable(flow_func):
         raise nx.NetworkXError("flow_func has to be callable.")
 
-    if kwargs.get("cutoff") is not None and flow_func in flow_funcs:
+    if kwargs.get("cutoff") is not None and flow_func in flow_funcs_without_cutoff:
```

Comment:
Related to the above comment: since we're down to one unsupported function, it might be more straightforward to check against that explicitly:

```suggestion
    if kwargs.get("cutoff") is not None and flow_func is preflow_push:
```